.github/workflows/nix.yml

```diff
@@ -38,12 +39,10 @@ jobs:
           signingKey: "${{ secrets.CACHIX_SIGNING_KEY }}"
       - name: Run integration tests
         run: make nix-integration-test
-      - name: Set up Go
-        uses: actions/setup-go@v2
-        with:
-          go-version: 1.20.0
       - name: Convert coverage data
-        run: go tool covdata textfmt -i=coverage -o profile.txt
+        run: |
+           nix profile install nixpkgs#go_1_20
```

Comment:
you might need to specify `nixpkgs` channel in `cachix/install-nix-action` to do this, not sure if it can run.

Reply:
oh, I guess it'll use the `flake.lock` in project directory?